icons/story.svg

```diff
@@ -0,0 +1,19 @@
+<svg
+  xmlns="http://www.w3.org/2000/svg"
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M12 2a10 10 0 0 1 7.38 16.75l-.06.06" />
+  <path d="M12 8v8" />
+  <path d="M16 12H8" />
+  <path d="M2.42 9.11a10 10 0 0 0-.42 3" />
+  <path d="m2.8 15.92.03.07a9.95 9.95 0 0 0 2.43 3.4" />
+  <path d="M4.1 5.88a9.449 9.449 0 0 1 .58-.68" />
+  <path d="M8.64 21.42A10.27 10.27 0 0 0 12 22a9.907 9.907 0 0 0 4.28-.96" />
```

Comment:
```suggestion
  <path d="M11.998 2a10 10 0 0 1 7.38 16.75" />
  <path d="M12 8v8" />
  <path d="M16 12H8" />
  <path d="M2.501 8.875a10 10 0 0 0-.5 3" />
  <path d="M2.829 15.986a10 10 0 0 0 2.43 3.4" />
  <path d="M4.636 5.235a10 10 0 0 1 .891-.857" />
  <path d="M8.644 21.42a10 10 0 0 0 7.631-.38" />
```

Reply:
Looks good and added. Just curious, but how did you refine the SVG? Looking to get better at making things pixel-perfect 🙏